Issue-2

LifeOfKnull opened the pull request:
- added dependencies for plugins (Components, Pages, Layouts, Scss)
- Added code snippets from github link
-  created pages folder with index
- verified app runs with test in app.vue

LifeOfKnull committed:
Issue-2
- added dependencies for plugins
(Components, Pages, Layouts, Scss)
- Added code snippets from github link
-  created pages folder with index
- verified app runs with test in app.vue

MichealPearce commented on client/src/App.vue:
```diff
@@ -1,45 +1,20 @@
+import { defineComponent } from 'vue'
+
+export default defineComponent({
+	name: 'test',
```
rename `App`

MichealPearce commented on client/src/App.vue:
```diff
@@ -1,45 +1,20 @@
-		</a>
-	</div>
-	<HelloWorld msg="Vite + Vue" />
+	<button class="test">this is a test</button>
```
Replace with `<RouterView />`

MichealPearce commented on client/src/App.vue:
```diff
@@ -1,45 +1,20 @@
-.logo.vue:hover {
-	filter: drop-shadow(0 0 2em #42b883aa);
+<style lang="scss" scoped>
+.test {
```
remove all styles for now

MichealPearce commented on client/vite.config.ts:
```diff
@@ -1,7 +1,19 @@
+	plugins: [
+		vue(),
+		Layouts(),
+		Pages(),
```
add `routeStyle` config for `Pages`, I believe it was `nuxt`

MichealPearce commented on package.json:
```diff
@@ -5,5 +5,8 @@
-	]
+	],
+	"devDependencies": {
+		"sass": "^1.56.1"
```
this needs to be moved to `client/package.json`

LifeOfKnull committed:
Issue #2
- Changed class name to app
- Changed template content to RouterView
- Removed Styling

LifeOfKnull committed:
Issue #2
- added nuxt routeType to vite.config.ts

LifeOfKnull committed:
Issue #2
- Removed dev dependency from main package,json file

LifeOfKnull committed:
Issue #2
- Renamed app to App

LifeOfKnull committed:
Issue #2
- Added layout folder
- Added RouterView to index
- Tested Index styling
